### DISCUSSION TITLE
Is it possible to generate entities programmatically instead of using `sea-orm-cli`?

### DISCUSSION THREAD

**Discussion by frederikhors** [SeaQL/sea-orm#1036] [Q&A] [answered]
Is it possible to generate entities programmatically instead of using `sea-orm-cli`?

I cannot find docs on this topic.

**Comment by tyt2y3** [comment, marked as answer]
`sea_orm_cli` is a library as well as an executable. So may be it has the API you need already exposed. https://docs.rs/sea-orm-cli/latest/sea_orm_cli/

**Comment by billy1624** [comment]
Yes, you can. Here is a old example for you
- https://github.com/SeaQL/sea-orm/pull/241
